app/scripts/views/mixins/password-strength-mixin.js

```diff
@@ -82,6 +82,7 @@ define(function (require, exports, module) {
             if (passwordCheckStatus === 'BLOOMFILTER_HIT' ||
                 passwordCheckStatus === 'BLOOMFILTER_MISS') {
               self._logStrengthExperimentEvent('bloomfilter_used');
+              self.displayErrorPrompt();
```

Comment:
this might need a better name `displayErrorPrompt`, should this be `displayPasswordNote` (or suggestion) or something?